crates/chain/tests/test_indexed_tx_graph.rs

```diff
@@ -469,5 +475,80 @@ fn test_list_owned_txouts() {
                 confirmed: 80000          // tx1 + tx3
             }
         );
+
+        // Test that not specifing the chain_tip return the same results
```

Comment:
025c4beef733ce76f45b5517e2ba895000d56de5 is adding some tests that 7adfedb992ab30b98edd03f08b54377e5505c73c is removing, can you squash the two commits together instead?